include/swift/Sema/ConstraintLocator.h

```diff
@@ -537,13 +557,13 @@ class StoredIntegerElement: public LocatorPathElt {
   /// Retrieve a value associated with the path element.
   template <unsigned Index = 0,
             typename = typename std::enable_if<(Index < NumValues)>::type>
-  unsigned getValue() const {
+  uint64_t getValue() const {
     // We pack values into 16 bit components of the storage, with value0
```

Comment:
This comment needs to be updated too because the packing now depends on number of values.

Reply:
Sure, done!